v2.3-RC2/station_information.json

```diff
@@ -128,6 +128,26 @@
                   }
                 }
               },
+              "parking_type": {
+                "description": "Type of parking station. Added in v2.3-RC2",
+                  "type": "string",
+                  "enum": [
+                    "parking_lot",
+                    "street_parking",
+                    "underground_parking",
+                    "sidewalk_parking",
+                    "other"
+                  ]
+                },
+              "parking_hoop": {
+                "description": "Are parking hoops present at this station? Added in v2.3-RC2",
+                "type": "boolean"
+              },
+              "contact_phone": {
+                "description": "Contact phone of the station. Added in v2.3-RC2",
+                "type": "string",
+                "pattern": "^\\+[1-9]\\d{1,14}$"
```

Comment:
For this version I don't think we can apply the pattern: regex to phone numbers. The spec says it's a field type of "Phone number" but the phone number field type isn't defined until v3.0 so it's just a string. The other field in this version where it's used (phone_number) allows numbers and letters.

Reply:
Oh, I assumed it was part of the V2.3-RC2 since it was part of the car-sharing Extention. But you're right, under the V2.3-RC2 label it is not appearing. Just fixed this